utilities/user_utilities.py

```diff
@@ -329,7 +330,7 @@ def approve_user_event(objectId):
         "$set": {"eventStatus": "approved"}
     })
     if not result:
-        __logger.error("Approve event {} fails in approve_event".format(id))
+        __logger.error("Approve event {} fails in approve_event. Status code {}.".format(id, result.status_code))
```

Comment:
can you also log the failure reason from the response as well?

Reply:
response.reason will give the text associated with the status code. Eg: OK for 200, Not Found for 404. Maybe logging the response.text would give more info?

Reply:
Added response.text in logs.. 
Let me know if response.reason should be included as well.

Reply:
Eg: Logging a successful PUT request
`__logger.info(
                    "Event {} PUT submission successful. Response {} {}".format(eventId, result.status_code, result.text))`

Would result in the below logs
```
2023-09-19T15:22:24.783Z INFO    [Thread-118] : user_utilities.py - Event 64f8eeac23a760827149006e PUT submission successful. Response 200 {
  "id": "64f8eeca940572fe64e2acb4",
  "message": "[PUT]: event id 64f8eeca940572fe64e2acb4, nUpdate = 1 ",
  "status": 200
}
```